Makefile

```diff
@@ -1,16 +1,16 @@
 .PHONY: all binary ut clean
 
 BUILD_DIR=build_calico_rkt
-BUILD_FILES=Dockerfile requirements.txt
+BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
```

Comment:
Why have you moved things to build_calico_rkt? It's only necessary in the main repo because we have two Dockerfiles, and it makes things more complex.

Standard Python project structure has requirements.txt at the root, and it's normal to have Makefile and Dockerfile at the root too.